modules/docs/mdx/welcome.stories.mdx

```diff
@@ -10,11 +9,10 @@ import {version} from '../../../lerna.json';
   parameters={{
     viewMode: 'docs',
     previewTabs: {
-      canvas: { hidden: true }
-     },
+      canvas: {hidden: true},
+    },
   }}
 />
-
```

Comment:
I've been looking into this for a few hours (somehow had node-gyp become a problem again and had to workaround) and it looks like removing this line results in the Welcome page not appearing anymore Storybook. This also happens on for the Changelog page in its respective story.

I was able to confirm this is the problem locally, and had to work around it either by disabling format on save or disabling Prettier. I couldn't find a setting to override to add this back in. Wrapping everything in a fragment didn't seem to help either.

before:
<img width="226" alt="Screenshot 2023-02-22 at 10 55 24 PM" src="https://user-images.githubusercontent.com/1075861/220830968-bef7bd35-2968-41ea-ab23-41ce0855e66c.png">

with the current change:
<img width="217" alt="Screenshot 2023-02-22 at 10 55 32 PM" src="https://user-images.githubusercontent.com/1075861/220831018-85fb1648-acaf-44f5-b117-e8cb67831938.png">

Reply:
That's an issue with MDX and prettier. Prettier removes the spacing between the JSX elements, but then MDX complains about needing a single react root. So we have to either add `<> </>` do them or separate them with a newline and save without formatting.